Ginger/GingerCore/Platforms/PlatformsInfo/JavaPlatform.cs

```diff
@@ -72,25 +72,70 @@ public override ObservableList<Act> GetPlatformElementActions(ElementInfo elemen
             ObservableList<Act> UIElementsActionsList = new ObservableList<Act>();
             eElementType elementType = GetElementType(elementInfo.ElementType);
 
-            var actionList = GetPlatformUIElementActionsList(elementType);
+            if (elementType.Equals(eElementType.Table))
+            {
+                //get all action list supported to tablecell action
+                var tableActionList = GetTableControlActions(ActUIElement.eElementAction.TableCellAction);
```

Comment:
instead of calling this method 3 times we can try to get it in one go using tuples like it is done in Web